src/inference/src/dev/core_impl.cpp

```diff
@@ -800,11 +798,57 @@ ov::AnyMap ov::CoreImpl::get_supported_property(const std::string& full_device_n
     } catch (ov::Exception&) {
     }
 
+    // if device_name represents virtual device (software plugin), we need to add properties for
```

Comment:
Need test case to cover corner cases like

1. core.compile_model(model, AUTO:GPU.1,GPU.0,CPU, {base_config, GPU_config, GPU_1_config}
2. AUTO load CPU with base_config
3. AUTO load GPU.0 with base_config+GPU_config. GPU_config is higher priority to overwrite base_config
4. AUTO load GPU.1 with base_config+GPU_config+GPU_1_config. GPU_1_config is higher priority to overwrite GPU_config

Reply:
All such tests are here https://github.com/openvinotoolkit/openvino/blob/master/src/inference/tests/unit/core.cpp#L199-L382